### PR TITLE
restructured information and added link to free tier tutorial

### PR DESCRIPTION
**Pull request by AnnikaGonnermann**

### CODE REVIEW

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md [hunk 1]
```diff
@@ -1,74 +1,75 @@
...
+
+- A **free trial account** for SAP BTP essentially provides you (as an individual) an isolated account and subaccount, with pre-allocated service entitlements for a limited amount of days.
+
+- **Free tier** is not an account but a set of entitlement plans within a consumption-based global account for activating certain services for free. So, say, you want to try out the SAP Launchpad tutorials, but in your company's existing global account -- you can create a new subaccount, activate the SAP Launchpad service in this subaccount using the free plan, create a service instance, and do the tutorials.
+
+    If you activate free tier plans only, you can try out our platform for free, without time limits. You can then easily upgrade to a paid plan without losing your data or work. Creating an instance using a paid plan will incur costs, however.
+
+    A plan limits the quantity of the service that's available.  Maybe you need a lot of some service in a production subaccount, and maybe you need less for development or test.  When you assign an entitlement to a subaccount, you also select the plan for use in that subaccount as well.
```
Comment (MichaelCzcz) [inline tutorials/btp-cockpit-setup/btp-cockpit-setup.md:25]: "...for develompent or test**ing**."

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/btp-cockpit-setup/btp-cockpit-setup.md (hunk 1 above)
Comment (AnnikaGonnermann) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md:25]: @MichaelCzcz: You are absolutely right!

Comment (MichaelCzcz) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md:25]: Did you make the change?

Comment (AnnikaGonnermann) [reply tutorials/btp-cockpit-setup/btp-cockpit-setup.md]: now yes :)

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Your branch has conflicts. I will merge it after you have resolved them.

**Comment by MichaelCzcz** [comment]
@AnnikaGonnermann If you don't resolve those conflicts, I can't merge your PR.

**Comment by AnnikaGonnermann** [comment]
@MichaelCzcz  I don't understand why Git shows the entire tutorial as one big conflict for me. I've deleted the conflict markers now, let's see what happens.

**Pull request closed without merging**